src/styles/index.css

```diff
@@ -0,0 +1,118 @@
+body {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+  font-weight: 400;
+}
+
+.card {
+  margin: 0 auto;
+  box-sizing: border-box;
+  width: 200px;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+}
+
+.card__title {
+  margin: 40px 0 4px;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+  color: #060b35;
+}
+
+.card__text {
+  margin: 0;
+  margin-bottom: 16px;
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 14px;
+  color: #616070;
+}
+
```

Comment:
you should use scss file to set styles
but now you are using css having scss files, it's weird
Keep only one format, for this task - scss